app/models/game.rb

```diff
@@ -1,11 +1,59 @@
 class Game < ActiveRecord::Base
-	has_many :users
+
+	has_and_belongs_to_many :user
```

Comment:
I think this is making the data model a bit more complex than needed.

I'd go with
```
has_one :white_player, class_name: :user
has_one :black_player, class_name: :user
```